app/views/devise/registrations/edit.html.slim

```diff
@@ -14,3 +14,10 @@
           =t('upaya.forms.registration.need_password')
         = f.input(:current_password, required: true)
         = f.button :submit, 'Update'
+      .hr
+      .panel-heading
+        h2.m0 = 'Authenticator App'
+      - if @update_user_profile_form.totp_enabled
+        = button_to 'Disable', disable_totp_path, method: :delete
+      - else
+        = link_to 'Enable', users_totp_path
```

Comment:
Should this be a button as well to keep the look consistent? This doesn't necessarily have to be addressed in this PR.